test/testaudioinfo.c

```diff
@@ -68,6 +70,34 @@ main(int argc, char **argv)
 
     print_devices(0);
     print_devices(1);
+    
+    
+    if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 0)) {
+        SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
+    } else {
+        SDL_Log("Default audio output info found!\n");
+#if MACOSX_COREAUDIO // Device name is unavailable for iOS/tvOS
```

Comment:
`MACOSX_COREAUDIO` is internal to SDL coreaudio source and isn't available to test programs: Are you actually seeing any output from this part of the test program on macosx?

Reply:
I would suggest initializing `deviceName` at top to NULL, and replace compile time check with a runtime check like `if(deviceName){...}`